### PR TITLE
Fix bug in resolving generic arrays.

### PR DESCRIPTION
**Pull request by MikeStall**

### CODE REVIEW

**Comment by mathewc** [comment]
I assume this needs to go out in the current batch of changes?

File: src/Microsoft.Azure.WebJobs.Host/Bindings/PatternMatcher.cs [hunk 1]
```diff
@@ -217,6 +224,21 @@ private static bool CheckArg(Type openType, Type specificType, Dictionary<string
                 return AddGenericArg(genericArgs, name, specificType);
             }
 
+            // T[]
+            if (openType.IsArray && specificType.IsArray)
+            {
+                var elementType = openType.GetElementType();
+                var specificElementType = specificType.GetElementType();
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Host/Bindings/PatternMatcher.cs]: nit: move this into if block

**Comment by MikeStall** [comment]
Merged into dev

File: src/Microsoft.Azure.WebJobs.Host/Bindings/PatternMatcher.cs (hunk 1 above)
Comment (MikeStall) [inline src/Microsoft.Azure.WebJobs.Host/Bindings/PatternMatcher.cs]: done

**Pull request closed without merging**